python_modules/libraries/dagster-sling/dagster_sling/dagster_sling_translator.py

```diff
@@ -60,6 +61,12 @@ def _resolve_back_compat_method(
         method = getattr(type(self), method_name)
         base_method = getattr(DagsterSlingTranslator, method_name)
         if method is not base_method:  # user defined this
+            supersession_warning(
+                subject=method_name,
+                additional_warn_text=(
+                    "Use `DagsterSlingTranslator.get_asset_spec(...)` to access the attribute instead."
```

Comment:
so this warning comes up specifically in cases where users are overriding the method on the base class, so maybe the warning should reference that? i.e. `Instead of overriding X, override DagsterSlingTranslator.get_asset_spec()`